src/diffusers/image_processor.py

```diff
@@ -80,7 +80,6 @@ def __init__(
                 " if you intended to convert the image into RGB format, please set `do_convert_grayscale = False`.",
                 " if you intended to convert the image into grayscale format, please set `do_convert_rgb = False`",
             )
-            self.config.do_convert_rgb = False
```

Comment:
why do we remove this?

Reply:
After the `raise`, would it be meaningful?

Reply:
@yiyixuxu, wouldn't it?